packages/inter-protocol/src/vpool-xyk-amm/doublePool.js

```diff
@@ -10,13 +10,13 @@ import {
 
 const { details: X } = assert;
 
-// Price calculations and swap using a pair of pools. Both pools map between RUN
-// and some collateral. We arrange the trades so collateralInPool will have
-// collateral added and collateralOutPool subtracted. When traders specify an
-// input price, that brand will be the inPool; when they specify the output
-// price that brand is the outPool.
-
 /**
+ * Price calculations and swap using a pair of pools. Both pools map between
+ * {Central} and some collateral. We arrange the trades so `collateralInPool`
+ * will have collateral added and `collateralOutPool` subtracted. When traders
+ * specify an input price, that brand will be the `inPool`; when they specify
+ * the output price that brand is the `outPool`.
+ *
```

Comment:
This feels like implementation doc rather than API description.  I think it makes more sense to move it outside the jsdoc. If you want some description for the jsdoc, I'd say

> doublePool is the virtualPool implementation for calculating prices when transiting through two pools. virtual pools wrap something that can do a swap and calculate prices.  Current wrapped pools are single pools and pairs of pools, but we've also contemplated using a similar virtual wrapper for other curves.

Reply:
done